### ISSUE TITLE
Boundary Issue

### ISSUE THREAD

**Issue by jqueryscript** [LyubomirT/moveme.js#1]
Seems like the right and bottom boundaries are not calculated properly.

**Comment by LyubomirT** [comment]
Yes, I just noticed this. Thanks for telling, should be fixed from now. Just note that you will have to use the version 1.1.0 instead of 1.0.7, since it's not fixed there.

If the issue still persists, please, let me know.
